website/client-old/js/services/guideServices.js

```diff
@@ -64,11 +64,15 @@ function($rootScope, User, $timeout, $state, Analytics) {
     classes: [
       [
         {
+          orphan: true,
+          content: window.env.t('classGearText'),
+          final: true,
           state: 'options.inventory.equipment',
           element: '.equipment-tab',
           title: window.env.t('classGear'),
-          content: window.env.t('classGearText')
-        }, {
+          hideNavigation: true
+        }
+        /*, {
```

Comment:
As far as I can tell, no; a quick git grep only shows the one call to 'classes' in userCtrl. See (Gist)[https://gist.github.com/cmajames/91d9469ed080589ebeba004f7c78b0e6].